apps/src/templates/sectionAssessments/sectionAssessmentsHelpers.js

```diff
@@ -0,0 +1,68 @@
+import commonMsg from '@cdo/locale';
```

Comment:
The file name should make it clear this is test data, something like `sectionAssessmentsTestHelpers`